lxd/instance/drivers/driver_qemu.go

```diff
@@ -6502,6 +6497,22 @@ func (d *qemu) MigrateSend(args instance.MigrateSendArgs) error {
 		return err
 	}
 
+	// Check for stateful support.
+	if args.Live && shared.IsFalseOrEmpty(d.expandedConfig["migration.stateful"]) {
```

Comment:
This still seems a bit of a workaround rather than a proper fix, why cant we send the control message error outside of d.MigrateSend() when it returns an error, otherwise we'll have to apply this fix for every error return in this function right?